UI/frontend-plugins/frontend-tools/data/scripts/instant-replay.lua

```diff
@@ -85,12 +105,14 @@ end
 -- A function named script_update will be called when settings are changed
 function script_update(settings)
 	source_name = obs.obs_data_get_string(settings, "source")
+	interval = obs.obs_data_get_int(setings, "interval")
+	max_attempts = obs.obs_data_get_int(setings, "attempts")
```

Comment:
I think that "attempts" should be "max_attempts" to keep it consistent with the script variable name.  If you change this here, make sure to change it throughout.